overlays/emacs.nix

```diff
@@ -126,6 +140,7 @@ let
       tree-sitter-json
       tree-sitter-tsx
       tree-sitter-typescript
+      tree-sitter-clojure
```

Comment:
Is there presently a tree-sitter-based Clojure major mode somewhere? If not, maybe we could drop this for now and add some kind of extra `withAllGrammars` option in some other commit.

Reply:
will drop it, there's no major mode yet.